pkgs/by-name/ta/taisei/package.nix

```diff
@@ -0,0 +1,102 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  # Build depends
+  docutils,
+  meson,
+  ninja,
+  pkg-config,
+  python3,
+  python312Packages,
+  ensureNewerSourcesForZipFilesHook,
+  # Runtime depends
+  glfw,
+  SDL2,
+  SDL2_mixer,
+  cglm,
+  freetype,
+  libpng,
+  libwebp,
+  libzip,
+  zlib,
+  zstd,
+  opusfile,
+  openssl,
+  gamemode,
+  shaderc,
+  spirv-cross,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "taisei";
+  version = "1.4.2";
+
+  src = fetchFromGitHub {
+    owner = "taisei-project";
+    repo = "taisei";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-rThLz8o6IYhIBUc0b1sAQi2aF28btajcM1ScTv+qn6c=";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [
+    docutils
+    meson
+    ninja
+    pkg-config
+    python3
+    python312Packages.zstandard
+    ensureNewerSourcesForZipFilesHook
+  ];
+
+  buildInputs = [
+    glfw
+    SDL2
+    SDL2_mixer
+    cglm
+    freetype
+    libpng
+    libwebp
+    libzip
+    zlib
+    zstd
+    opusfile
+    openssl
+    gamemode
+    shaderc
+    spirv-cross
+  ];
+
+  mesonFlags = [
```

Comment:
You could use `lib.mesonBool` and `lib.mesonEnable` here, though it's fine too keep it as is.

e.g.:
`(lib.mesonBool "b_lto" false)`
`(lib.mesonEnable "install_macos_bundle" false)`